mailserver.env

```diff
@@ -479,3 +479,16 @@ RELAY_USER=
 # empty => no default
 # password for default relay user
 RELAY_PASSWORD=
+
+# -----------------------------------------------
+# --- Security ----------------------------------
+# -----------------------------------------------
+
+# Set up per user storage encryption with passworded keys
+#
+# empty => disabled
```

Comment:
```suggestion
# 0 => disabled
```

Reply:
As you set 
```
VARS[ENABLE_PER_USER_STORAGE_ENCRYPTION]="${ENABLE_PER_USER_STORAGE_ENCRYPTION:=0}"
```

Probably better keep it with `empty => disabled` for a consistent style with the rest of the file and ignore this change request